pymodels/BO_V06_01/accelerator.py

```diff
@@ -25,8 +28,23 @@ def create_accelerator(
     return accelerator
 
 
+lattice_version = 'BO_V06_01'
```

Comment:
~~all this should come before you create the accelerator.~~
Actually I don't understand how this works, because you created a local variable named `lattice_version` inside the `create_accelerator` function and also another module variable with the same name and both of them have different values.

Reply:
I had forgotten to delete this local variable definition in `create_accelerator`, thanks!